views/menus/tabs.js

```diff
@@ -36,14 +36,14 @@ const stolaf = [
     id: 'pause',
     title: 'The Pause',
     rnVectorIcon: {iconName: 'paw'},
-    component: PauseMenuView,
-    // props: {
-    //   name: 'pause',
-    //   loadingMessage: [
-    //     'Mixing up a shake…',
-    //     'Spinning up pizzas…',
-    //   ],
-    // },
+    component: GitHubHostedMenu,
+    props: {
+      name: 'pause',
+      loadingMessage: [
+        'Mixing up a shake…',
+        'Spinning up pizzas…',
+      ],
+    },
```

Comment:
@drewvolz This is why I left those commented in #452 – thinking it might make the diff smaller.